src/sentry/statistical_detectors/issue_platform_adapter.py

```diff
@@ -56,6 +57,7 @@ def send_regression_to_platform(regression: BreakpointData):
         ],
         detection_time=current_timestamp,
         level="info",
+        initial_issue_priority=PriorityLevel.MEDIUM,
```

Comment:
@malwilley @snigdhas How does this interact with the `level` property above? Will that be ignored now, or should that also be updated to something higher?